zstacklib/zstacklib/utils/fileUploader.py

```
@@ -0,0 +1,289 @@
+        b = b.strip()
+        if b == ib:
+            break
+
+    return ib
+
+
+def stream_body(entity, boundary, param, task):
+    # type: (Entity, str, UploadParam, UploadTask) -> None
+
+    _, image_name = task.tmpPath.split('/')
+    task.create_image_if_not_exists(ioctx, image_name)
```

Ruff (0.13.1)

`ioctx` may be undefined, or defined from star imports

(F405)